src/pulsar_consumer.erl

```diff
@@ -49,6 +52,10 @@ callback_mode() -> [state_functions].
 start_link(PartitionTopic, Server, ProxyToBrokerUrl, ConsumerOpts) ->
     gen_statem:start_link(?MODULE, [PartitionTopic, Server, ProxyToBrokerUrl, ConsumerOpts], []).
 
+-spec get_state(pid()) -> statem().
+get_state(Pid) ->
```

Comment:
Perhaps it could be implemented using `sys:get_state`?

Reply:
In the [bridge tests](https://github.com/emqx/emqx/pull/10378/files#diff-3f42ee75729c76edb520109b17910ba6ead94bcbfa98321a501b6177e2803497R436), I ended up doing that for simplicity (since I don't have easy direct access to the pids).  :joy_cat: 

Do you think it would be better to switch this one to use `sys:get_state`?